packages/ember/index.js

```diff
@@ -171,7 +171,7 @@ import {
 const Ember = (typeof context.imports.Ember === 'object' && context.imports.Ember) || {};
 
 Ember.isNamespace = true;
-Ember.toString = function () {
+Ember.toString = function() {
```

Comment:
you'll want to roll back these unrelated prettier changes (also seems like maybe your editor was applying a different prettier version than our linting uses)

Reply:
for sure, idk what happened here (very WIP PR) :p